internal/tiltfile/tiltfile_state.go

```diff
@@ -1036,6 +1037,7 @@ func (s *tiltfileState) translateK8s(resources []*k8sResource) ([]model.Manifest
 			Name:                 mn,
 			TriggerMode:          tm,
 			ResourceDependencies: mds,
+			NonWorkload:          r.nonWorkload,
```

Comment:
can you add a unit test in internal/tiltfile that demonstrates how this is set? I'm not totally sure I understand the difference between "a manifest with no image targets" and "a manifest with no workloads"

Reply:
Yeah it's possible that this is unnecessary, let me do some more digging.

Reply:
ping this comment; i think having a test for when we expect this to be true to be the important bit

Reply:
Added some assertions to the tiltfile tests @nicks, thanks for pointing this out